javascript-data-model/order-history.js

```diff
@@ -4,44 +4,56 @@ var orderHistory = [
     total: '$34.00',
     ship: 'JS Masher',
     orderNumber: '114 - 3941689 - 8772232',
-    type: 'book',
-    title: 'Javascript for impatient programmers',
-    delivered: 'Aug 8, 2020',
+    deliveredDate: 'Aug 8, 2020',
     reviewed: false,
     buyAgain: false,
     returnWindowClosed: true,
     returnWindowDate: 'Sept 7, 2020',
-    itemPrice: '$31.55'
+    item: [
+      {
+        type: 'book',
+        title: 'Javascript for impatient programmers',
+        author: 'Rauschmayer, Dr. Axel',
+        itemPrice: '$31.55'
+      }
+    ]
   },
   {
     ordered: 'July 19, 2020',
     total: '$44.53',
     ship: 'JS Masher',
     orderNumber: '113-9984268-1280257',
-    type: 'book',
-    title: 'The Timeless Way of Building',
-    delivered: 'Jul 20, 2020',
+    deliveredDate: 'Jul 20, 2020',
     reviewed: false,
     buyAgain: false,
     returnWindowClosed: true,
     returnWindowDate: 'Aug 19, 2020',
-    itemPrice: '$41.33'
+    item: [
+      {
+        type: 'book',
+        title: 'The Timeless Way of Building',
+        itemPrice: '$41.33'
+      }
+    ]
   },
 
   {
     ordered: 'July 4, 2020',
     total: '$17.22',
     ship: 'JS Masher',
     orderNumber: '114-2875557-9059409',
-    type: 'console',
-    title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube  Adapter for WII UIEvent PC. Support Turbo and Vibration Feaures. No Driver and No Lag Gamecube Adapter',
-    delivered: 'Jul 07, 2020',
+    deliveredDate: 'Jul 07, 2020',
     reviewed: false,
     buyAgain: false,
     returnWindowClosed: true,
     returnWindowDate: 'Aug 05, 2020',
-    itemPrice: '$15.98'
-
+    item: [
```

Comment:
You have the property named `item` here, but it is `items` in your final order object. You should rename all of the properties storing the items ordered to `items`, even if there is only one in the actual data.